source/Engine.cpp

```diff
@@ -2765,15 +2783,19 @@ void Engine::DrawShipSprites(const Ship &ship)
 			if(bay.side == Ship::Bay::UNDER && bay.ship)
 				drawObject(*bay.ship);
 
-	if(ship.IsThrusting() && !ship.EnginePoints().empty())
+	double scaleMul = ship.Zoom() * Ship::THRUST_HELD_FRAMES_RECIP;
+	Point thrustMul = FlareCurve(ship.ThrustHeldFrames(Ship::ThrustKind::FORWARD) * scaleMul);
```

Comment:
When you call `FlareCurve` from `DrawFlareSprites` to get the scales for the left and right turning flares, you don't multiply the value you give it by `Ship::Zoom()`, but you do here. Why the difference?

I think it would be better to just replace `ThrustHeldFrames` with a method that multiplies by `THRUST_HELD_FRAMES_RECIP` and then returns the result; that would also remove the need for the public static class fields in Ship.